src/multipatch/connectivity/edge_transformation.hpp

```diff
@@ -269,16 +423,27 @@ class EdgeTransformation
                 target_idx += target_idx_step_rescaled;
 
                 if constexpr (!Interface::orientations_agree) {
-                    IdxStepTarget target_idx_step = IdxTarget(target_idx_range.back()) - target_idx;
-                    target_idx = IdxTarget(target_idx_range.front());
-                    target_idx += target_idx_step;
+                    if constexpr (TargetGrid::continuous_dimension_type::PERIODIC) {
+                        IdxStepTarget target_idx_step
+                                = IdxTarget(target_idx_range.back() + 1) - target_idx;
```

Comment:
I am a little lost in these calculations. Is it not simpler to calculate the `target_idx_step_rescaled` and then calculate `target_idx` in the `if constexpr`:
```cpp
if constexpr (!Interface::orientations_agree) {
    target_idx = target_idx_range.back() + int(TargetGrid::continuous_dimension_type::PERIODIC) - target_idx_step_rescaled;
} else {
    target_idx = target_idx_range.front() + target_idx_step_rescaled;
}
```
?